Source/WebCore/platform/graphics/gstreamer/eme/WebKitCommonEncryptionDecryptorGStreamer.cpp

```diff
@@ -205,14 +205,27 @@ static GstFlowReturn webkitMediaCommonEncryptionDecryptTransformInPlace(GstBaseT
     WebKitMediaCommonEncryptionDecrypt* self = WEBKIT_MEDIA_CENC_DECRYPT(base);
     WebKitMediaCommonEncryptionDecryptPrivate* priv = WEBKIT_MEDIA_CENC_DECRYPT_GET_PRIVATE(self);
 
+    GstProtectionMeta* protectionMeta = reinterpret_cast<GstProtectionMeta*>(gst_buffer_get_protection_meta(buffer));
+    bool isBufferEncrypted = protectionMeta;
+
     LockHolder locker(priv->m_mutex);
 
+    if (protectionMeta) {
+        const GValue* streamEncryptionEventsList = gst_structure_get_value(protectionMeta->info, "stream-encryption-events");
```

Comment:
I reckon that it can be more intuitive if you use gst_structure_get instead of get_value but since we are going to use values here I think my approach is more efficient because with the get_value call the checks for the existance are going to be performed anyway and bail out with a null so we would be checking twice. This way we only check once inside get_value and assess the result and the type if available.

Reply:
OK, was just a minor comment, I'm happy either way :+1: